packages/datadog_flutter_plugin/lib/src/rum/ddrum.dart

```diff
@@ -96,6 +96,28 @@ class DdRum {
 
   RumLongTaskObserver? _longTaskObserver;
 
+  /// Get the last active session id, or an empty string if a session hasn't started
+  ///
+  /// Session ids are updated when a new session is started in Datadog, usually
+  /// as a result of a user action. If you are using [stopSession], [sessionId] will
+  /// continue to return the last active session until a new session starts.
```

Comment:
Shouldn't we return `null` instead of there is no active session? what is the reasoning behind returning the last active session when it was stopped? What Browser SDK is doing in this case?